client/src/components/Login.js

```diff
@@ -14,7 +14,7 @@ const Login = () => {
   useEffect(() => {
     const fetchProfile = async applicantId => {
       const response = await getProfileByApplicantId(applicantId);
-      if (Object.keys(response).length !== 0) {
+      if (response.length !== 0) {
```

Comment:
This could change to `!response.length` instead of `response.length !== 0`

Reply:
Changed. Thanks for your assistance.